modules/osdk-run-proxy.adoc

```diff
@@ -0,0 +1,155 @@
+//
+// * operators/operator_sdk/golang/osdk-golang-tutorial.adoc
+// * operators/operator_sdk/ansible/osdk-ansible-tutorial.adoc
+// * operators/operator_sdk/helm/osdk-helm-tutorial.adoc
+
+ifeval::["{context}" == "osdk-golang-tutorial"]
+:golang:
+:app-proper: Memcached
+:app: memcached
+:group: cache
+endif::[]
+ifeval::["{context}" == "osdk-ansible-tutorial"]
+:ansible:
+:app-proper: Memcached
+:app: memcached
+:group: cache
+endif::[]
+ifeval::["{context}" == "osdk-helm-tutorial"]
+:helm:
+:app-proper: Nginx
+:app: nginx
+:group: demo
+endif::[]
+
+[id="osdk-run-proxy_{context}"]
+
+= Proxy support
+
+To support proxied clusters, your Opertator must inspect the environment for the following standard proxy variables and pass the values to Operands:
+
+- `HTTPS_PROXY`
+- `HTTPS_PROXY`
+- `NO_PROXY`
+
+
+ifdef::golang[]
+Add the `proxy.ReadProxyVarsFromEnv` helper function to the reconcile loop in the `controllers/memcached_controller.go` file and append the results to the Operand environments:
+
+[source,golang,subs="attributes+"]
+----
+for i, container := range dep.Spec.Template.Spec.Containers {
+		dep.Spec.Template.Spec.Containers[i].Env = append(container.Env, proxy.ReadProxyVarsFromEnv()...)
+}
+----
+
+endif::[]
+
+// ifdef::ansible[]
+// To setup your Operator to scan the deployment environment,
+// +
+// [source,yaml,subs="attributes+"]
+// ----
+// - name: start proxy-test job
+//   kubernetes.core.k8s:
+//     definition:
+//       apiVersion: batch/v1
+//       kind: Job
+//       metadata:
+//         name: env-job
+//         namespace: "{{ ansible_operator_meta.namespace }}"
+//       spec:
+//         template:
+//           spec:
+//             containers:
+//               - name: curl-example
+//                 image: registry.access.redhat.com/ubi8/ubi:8.4-209
+//                 command: ["curl"]
+//                 args: ["http://example.com/job-request"]
+//                 env:
+//                   - name: HTTP_PROXY
+//                     value: '{{ lookup("env", "HTTP_PROXY") | default("", True) }}'
+//                   - name: http_proxy
+//                     value: '{{ lookup("env", "HTTP_PROXY") | default("", True) }}'
+//             restartPolicy: Never
+//         backoffLimit: 4
+// ----
+
+// endif::[]
+
+ifdef::helm[]
+To include the overrides based on an environment varialbe, modify the `watches.yaml` file to include:
+
+[source,yaml,subs="attributes+"]
+----
+- group: demo.example.com
+  version: v1alpha1
+  kind: Nginx
+  chart: helm-charts/nginx
+  overrideValues:
+    proxy.http: $HTTP_PROXY
+#+kubebuilder:scaffold:watch
+----
+
+Add the `proxy.http` value in the `helmcharts/nginx/Values.yaml` file:
+
+[source,yaml,subs="attributes+"]
+----
+proxy:
+  http: ""
+  https: ""
+  no_proxy: ""
+----
+
+To make sure the chart template supports using the varialbes, edit the chart template in the `helm-charts/nginx/templates/deployment.yaml` file to contain the following:
```

Comment:
```suggestion
To make sure the chart template supports using the variables, edit the chart template in the `helm-charts/nginx/templates/deployment.yaml` file to contain the following:
```

And again in the example it was nginx but in a users operator it could be something different. Maybe generically mention the `deployment.yaml` template.